### DISCUSSION TITLE
Remote machine learning: do I need to move the model cache after the initial import?

### DISCUSSION THREAD

**Discussion by dekiesel** [immich-app/immich#5872] [Q&A] [answered]
Hi,

I have immich running on a minipc. For my initial load I used another, bigger, machine so that tagging and face recognition and so on would be quicker.

Now that I am done, can I just spin up another ml container on the minipc or do I need to move the ml model cache to the minipc too?

Thanks!

**Comment by bo0tzz** [comment, marked as answer]
The cache is just for the model files; if those are not present, they will be downloaded again, so you're fine to spin up a new container without moving the cache.

**Reply by dekiesel** [reply]
Great, thanks for the answer! :)
